lib/node_modules/@stdlib/stats/incr/wstdev/docs/repl.txt

```diff
@@ -0,0 +1,31 @@
+{{alias}}( [mean] )
```

Comment:
This file does not follow our REPL text conventions and should be updated accordingly. See the REPL text style guide.